hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/parse/UnsupportedOperationVisitor.java

```diff
@@ -154,6 +161,16 @@ public Void visit(SqlNodeList nodeList) {
 
     @Override
     public Void visit(SqlIdentifier id) {
+        SqlValidatorTable table = catalogReader.getTable(id.names);
```

Comment:
What is the purpose of this whole check?

Reply:
For instance `SELECT * FROM kafka` should fail, even if all operators are supported by IMDG, it does not mean it can handle such statement